x-pack/plugin/inference/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/xpack/application/CohereServiceUpgradeIT.java

```diff
@@ -169,7 +174,8 @@ void assertEmbeddingInference(String inferenceId, CohereEmbeddingType type) thro
     }
 
     @SuppressWarnings("unchecked")
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/107887")
+    // TODO: mute again afterwards
+    // @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/107887")
```

Comment:
reminder: mute this test even, if it works again to keep re-enabling separate and isolated from this PR